charon/tests/cli.rs

```diff
@@ -69,3 +69,45 @@ fn charon_cargo_p_crate2() -> Result<()> {
         },
     )
 }
+
+#[test]
+fn charon_cargo_features() -> Result<()> {
```

Comment:
This is redundant with the existing `cargo/dependencies` test. I'd prefer if you just updated the command-line instruction we use for that test in `cargo.rs`.

I prefer tests that commit the whole output because imo tests like this one that test for the presence/absence of a string are more annoying to fix when they break.

Reply:
You know what, I think your way is more robust actuallyl, nevermind.